js/genkit/src/genkit.ts

```diff
@@ -308,9 +308,7 @@ export class Genkit {
   }
 
   /**
-   * Defines and registers a dotprompt.
-   *
-   * This is an alternative to defining and importing a .prompt file.
+   * Defines and registers a function-based prompt.
```

Comment:
I just swapped the order of these signatures... helps VSCode.... not sure why....